### DISCUSSION TITLE
WASM Example

### DISCUSSION THREAD

**Discussion by ixxie** [uwdata/mosaic#170] [Q&A] [answered]
Hey folks,

New user here; I saw in your README that `mosaic-core` can be setup to run with a DuckDB-WASM instance. 

Do you have any examples of such a setup? 

I'm finding it a bit hard to figure out from the docs how to put something like that together.

Thanks for making this awesome piece of software! I'm excited to get hacking with it :)

**Comment by domoritz** [comment, marked as answer]
The WASM connector setup for the development demo is at https://github.com/uwdata/mosaic/blob/6a470c4b56946ad7e85fc02b463ea4eb7b8e6441/dev/setup.js#L19. As you can see, the setup for any connector is almost the same as the other connectors.

**Comment by ixxie** [comment]
Thank you @domoritz; can you elaborate on the purpose of the development demo? I haven't seen it on the documentation site...

**Reply by domoritz** [reply]
It's not documented. It's basically a small web app we use to test all the examples. You need to run `npm run server` and `npm run dev` in separate terminal sessions and then can test the examples (YAML and JS) against the node server and WASM.
